components/properties/src/props.rs

```diff
@@ -891,6 +897,14 @@ impl Script {
 }
 }
 
+impl Script {
+    // Doesn't actually exist!
+    #[doc(hidden)]
+    #[allow(non_upper_case_globals)]
+    #[deprecated]
+    pub const Chisoi: Script = Self(254);
```

Comment:
Chisoi was removed from Unicode 17 late in the cycle. It will probably be in Unicode 18, so it might have a value like 211. Question is: do we gamble on it becoming 211, or do we assign 254 and then later change it again? Or do we actually break semver and remove it?

Reply:
I vote to break semver and release this as 2.1.1 asap before people start needing it.